WeScan/Scan/ScannerViewController.swift

```diff
@@ -73,25 +73,40 @@ final class ScannerViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        title = NSLocalizedString("wescan.scanning.title", tableName: nil, bundle: Bundle(for: ScannerViewController.self), value: "Scanning", comment: "The title of the ScannerViewController")
+        title = nil
         
         setupViews()
-        setupToolbar()
+        setupNavigationBar()
         setupConstraints()
         
         captureSessionManager = CaptureSessionManager(videoPreviewLayer: videoPreviewLayer)
         captureSessionManager?.delegate = self
         
+        originalBarStyle = navigationController?.navigationBar.barStyle
+        
         NotificationCenter.default.addObserver(self, selector: #selector(subjectAreaDidChange), name: NSNotification.Name.AVCaptureDeviceSubjectAreaDidChange, object: nil)
     }
     
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
+        setNeedsStatusBarAppearanceUpdate()
+        
         CaptureSession.current.isEditing = false
         quadView.removeQuadrilateral()
         captureSessionManager?.start()
         UIApplication.shared.isIdleTimerDisabled = true
-        navigationController?.setNavigationBarHidden(true, animated: false)
+        
+        let statusBarHeight = UIApplication.shared.statusBarFrame.size.height
+        let visualEffectRect = self.navigationController?.navigationBar.bounds.insetBy(dx: 0, dy: -(statusBarHeight)).offsetBy(dx: 0, dy: -statusBarHeight)
+        
+        visualEffectView.frame = visualEffectRect ?? CGRect.zero
+
+        navigationController?.navigationBar.isTranslucent = true
+        navigationController?.navigationBar.setBackgroundImage(UIImage(), for: .default)
+        navigationController?.navigationBar.addSubview(visualEffectView)
```

Comment:
We should probably not add subviews in `viewWillAppear` since it may be called multiple times. `viewDidLoad` seems more appropriate.

Reply:
Hey, this is because we don't want the blur effect to be shown on other screens. It's removed in `viewWillDisappear`.

Reply:
Don't we add it twice right now? On `viewDidLoad` via `setupNavigationBar` and here on `viewWillAppear`?

Reply:
Fixed!